src/Effects.js

```diff
@@ -191,5 +191,72 @@ module.exports = class Effects {
         }
       }
     };
+    this.splatter = {
+      splats:[],
+      numSplats:100,
+      randomSplat: function () {
+        let r = randomNumber(30,60);
+        return {x: randomNumber(0,400),
+            y: randomNumber(0,400),
+            color: p5.color("hsl(" + randomNumber(0, 359) + ", 100%, 80%)"),
+            width: r,
+            height: r,
+        };
+      },
+      init: function () {
+        for (var i=0;i<this.numSplats;i++) {
+          this.splats.push(this.randomSplat());
+        }
+        p5.strokeWeight(0);
+      },
+      update: function () {
+        //TODO: add some music-driven change? Right now it just grows continuously.
+      },
+      draw: function () {
+        if (this.splats.length<1) {
+          this.init();
+        }
+        p5.svg();
+        p5.strokeWeight(0);
+        for (var i=0;i<this.splats.length;i++) {
+          if (randomNumber(0,50) === 0) {
+            this.splats[i]=this.randomSplat();
+          }
+          p5.fill(this.splats[i].color);
+          this.splats[i].width+=randomNumber(0,4);
+          this.splats[i].height+=randomNumber(0,4);
+          p5.ellipse(this.splats[i].x,this.splats[i].y,this.splats[i].width,this.splats[i].height);
+        }
+      }
+    };
+    this.spiral = {
+      swirl: null,
+      angle: 0,
+      color: null,
+      init: function () {
+        //I should do this somewhere else, right?
+        this.swirl = p5.loadImage("https://curriculum.code.org/images/spiral.svg");
```

Comment:
I wonder if this is a small enough asset that we should just include it inline?

Sample: https://editor.p5js.org/joshlory/sketches/HkH6IjRjm